book/cli/reth/node.md

```diff
@@ -323,7 +323,7 @@ RPC:
           Set the maximum RPC response payload size for both HTTP and WS in megabytes
 
           [default: 160]
-          [aliases: rpc.returndata.limit]
+          [aliases: --rpc.returndata.limit]
```

Comment:
where's this coming from

Reply:
idk...form the https://github.com/paradigmxyz/reth/pull/16256 PR, clueless, CI failing without this fix on that PR https://github.com/paradigmxyz/reth/actions/runs/15067101679/job/42355125495

Reply:
nope

Reply:
ah on https://github.com/paradigmxyz/reth/pull/16256 could be, yes, thx